MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -7077,5 +7087,80 @@ private void ul2as()
         ///     Signature: int errcod;
         /// </summary>
         private ReadOnlySpan<byte> errcod => Module.Memory.GetVariablePointer("ERRCOD").Data;
+
+        /// <summary>
+        ///     Determines the profanity level of a string
+        ///
+        ///     MBBSEmu doesn't support multiple profanity levels, so the default value of 0 is returned.
+        /// 
+        ///     Signature:  int profan(char *string);
+        /// </summary>
+        private void profan()
+        {
+            Registers.AX = 0;
+        }
+
+        /// <summary>
+        ///     Expect a YES or NO from the user
+        ///
+        ///     Signature: int yesno=cncyesno();
+        /// </summary>
+        private void cncyesno()
+        {
+            //Get Input
+            var inputLength = Module.Memory.GetWord("INPLEN");
+            var inputPointer = Module.Memory.GetVariablePointer("INPUT");
+            var inputString = Module.Memory.GetArray(inputPointer, inputLength);
+            var inputStringComponents = Encoding.ASCII.GetString(inputString).ToUpper().Split('\0');
```

Comment:
shouldn't this operate on the current value of NXTCMD rather than the start of INPUT?

Reply:
Right you are! Need to update the method -- and probably revisit the previous CNC* methods.

Reply:
Updated code to operate off current pointer of NXTCMD in INPUT. Updated tests to include scenarios where NXTCMD.Offset > INPUT.Offset